src/app/tests/suites/certification/Test_TC_DRLK_2_6.yaml

```diff
@@ -107,40 +107,38 @@ tests:
           error: INVALID_COMMAND
 
     - label:
-          "Step 5: TH sends Get Holiday Schedule Command to DUT with Invalid
-          HolidayIndex as 15"
+          "Step 5: TH sends GetHolidaySchedule Command to DUT with Invalid
+          HolidayIndex > number_holiday_sch_supported ."
       PICS: DRLK.S.F0b && DRLK.S.C12.Rsp && DRLK.S.C12.Tx
       command: "GetHolidaySchedule"
       arguments:
           values:
               - name: "HolidayIndex"
-                value: 15
+                value: NumberOfHolidaySchedulesSupportedValue + 1
       response:
           values:
-              - name: "HolidayIndex"
-                value: 15
```

Comment:
So...  The spec says two things that are mutually contradictory here:

1. HolidayIndex SHALL be set to the value in the request command (even if that's out of range, as far as I can tell).
2. HolidayIndex constraint is "1 to NumberOfHolidaySchedulesSupported".

We need to get the spec fixed here, probably by fixingthat constraint to be something like "all".

In the meantime, the test plan should in fact be checking that the value that came back is `NumberOfHolidaySchedulesSupportedValue + 1`, since the spec does require that.

Reply:
I think you are referring to the HolidayIndex in the GetHolidaySchedule response -- and the specific thing you're asking is: IF the GetHolidaySchedule request includes a HolidayIndex that is invalid, what shall the HolidayIndex be in the response...

I'm not sure how the SDK currently behaves, but IMO what should be done here is to change the spec for the HolidayIndex in the **response** to be **all**

Reply:
That would be consistent with the SDK behavior, yes.  The SDK just echoes back whatever index came in, including if it's invalid.